COLLABORATOR_GUIDE.md

```diff
@@ -795,9 +792,9 @@ If you cannot find who to cc for a file, `git shortlog -n -s <file>` may help.
 [`--throw-deprecation`]: doc/api/cli.md#--throw-deprecation
 [`node-core-utils`]: https://github.com/nodejs/node-core-utils
 [backporting guide]: doc/guides/backporting-to-release-lines.md
-[contributing]: ./doc/guides/contributing/pull-requests.md#commit-message-guidelines
+[commit message guidelines]: ./doc/guides/contributing/pull-requests.md#commit-message-guidelines
 [commit-example]: https://github.com/nodejs/node/commit/b636ba8186
-[flaky tests]: https://github.com/nodejs/node/issues?q=is%3Aopen+is%3Aissue+label%3A%22CI+%2F+flaky+test%22y
+[unreliable tests]: https://github.com/nodejs/node/issues?q=is%3Aopen+is%3Aissue+label%3A%22CI+%2F+flaky+test%22y
```

Comment:
This needs to be moved to the end of the references list now.

Reply:
Moved. Thanks!